Mathlib/NumberTheory/NumberField/InfinitePlace/Basic.lean

```diff
@@ -93,6 +93,21 @@ theorem norm_embedding_eq (w : InfinitePlace K) (x : K) :
   nth_rewrite 2 [← mk_embedding w]
   rfl
 
+variable (K) in
+theorem embedding_injective : (embedding (K := K)).Injective := by
+   intro v₁ v₂ h
+   rw [← mk_embedding v₁, h, mk_embedding]
+
+variable (K) in
+theorem conjugate_embedding_injective :
+     (fun (v : InfinitePlace K) => ComplexEmbedding.conjugate v.embedding).Injective :=
+   star_injective.comp <| embedding_injective K
+
+variable (K) in
+theorem eq_of_embedding_eq_conjugate {v₁ v₂ : InfinitePlace K}
+     (h : v₁.embedding = ComplexEmbedding.conjugate v₂.embedding) : v₁ = v₂ := by
+   rw [← mk_embedding v₁, h, mk_conjugate_eq, mk_embedding]
```

Comment:
Please follow mathlib indentation style - four spaces for the signature and two for the proof